src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -0,0 +1,170 @@
+        :return: summary
+        """
+        # start threads in MSManager which should call this method
+        # NOTE:
+        #    Here we should make the whole logic - like:
+        #    * Calling the system to fetch the workflows from;
+        #    * Creating the workflow objects;
+        #    * Pushing them into the back end database system we choose for bookkeeping
+        #    * Updating their status in that system, both MsStatus (subscribed,
+        #      processing, etc.) and also the Reqmgr status
+        #    * Associate and keep track of the requestID/subscriptionID/ruleID
+        #      returned by the Data Management System and the workflow
+        #      object (through the bookkeeping machinery we choose/develop)
+        summary = dict(OUTPUT_REPORT)
+
+        # We should decide here what variable name to use request vs. workflow
+        try:
+            requestRecords = self.getRequestRecords(reqStatus)
+            self.updateReportDict(summary, "total_num_requests", len(requestRecords))
+            msg = "  retrieved %s requests. " % len(requestRecords)
+            msg += "Service set to process up to %s requests per cycle." % self.msConfig["limitRequestsPerCycle"]
+            self.logger.info(msg)
+        except Exception as err:  # general error
+            msg = "Unknown exception while fetching requests from ReqMgr2. Error: %s", str(err)
+            self.logger.exception(msg)
+            self.updateReportDict(summary, "error", msg)
+
+        try:
+            self.updateCaches()
+        except RuntimeWarning as ex:
+            msg = "All retries exhausted! Last error was: '%s'" % str(ex)
+            msg += "\nRetrying to update caches again in the next cycle."
+            self.logger.error(msg)
+            self.updateReportDict(summary, "error", msg)
+            return summary
+        except Exception as ex:
+            msg = "Unknown exception updating caches. Error: %s" % str(ex)
+            self.logger.exception(msg)
+            self.updateReportDict(summary, "error", msg)
+            return summary
+
+        # this one is put here just for example.
+        self.updateReportDict(summary, "ddm_request_id", 42)
+        return summary
+
+    def makeSubscriptions(self, workflows=[]):
+        """
+        The common function to make the final subscriptions. It depends on the
+        default Data Management System configured through msConfig. Based on that
+        The relevant service wrapper is called.
+        :return: A list of results from the REST interface of the DMS in question
+        """
+
+        # NOTE:
+        #    Here is just an example construction of the function. None of the
+        #    data structures used to visualise it is correct. To Be Updated
+        results = []
+        if self.msConfig['defaultDataManSys'] == 'DDM':
+            # TODO: Here to put the dryrun mode: True/False
+            ddm = DDM(
```

Comment:
This should be initialized in the `__init__` method. Here we check which DM system to use and make the proper API call.